### ISSUE TITLE
Epic: Show the Charges tile on the Payment Activity widget for the store’s primary currency and Last 7 days

### ISSUE THREAD

**Issue by jessy-p** [Automattic/woocommerce-payments#8486]
On click of the Charges, redirects to Transactions page filtered by date(Last 7 days) for Charges . This click is recorded by track - track event name TBD
```[tasklist]
### Tasks
- [ ] https://github.com/Automattic/woocommerce-payments/issues/8705
```

**Comment by nagpai** [comment]
Task #8595 - Includes adding UI for the Charges tile . This task is already included in Epic #8484 , hence not added as a task here.

**Comment by shendy-a8c** [comment]
@nagpai, that means this issue can be closed right? I'll close it for now.
